pkgs/data/fonts/openmoji/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, scfbuild
+, python3Packages
+, variant ? "color" # "color" or "black"
+}:
+
+let
+  filename = builtins.replaceStrings
+    [ "color"              "black"              ]
+    [ "OpenMoji-Color.ttf" "OpenMoji-Black.ttf" ]
+    variant;
+
+in stdenv.mkDerivation rec {
+  pname = "openmoji";
+  version = "12.4.0";
+
+  src = fetchFromGitHub {
+    owner = "hfg-gmuend";
+    repo = pname;
+    rev = version;
+    sha256 = "0zis0i7fn7dqv39c9ms7f1p4dqqzwdfwkdgrijaak1cmb8rkn3a0";
+  };
+
+  nativeBuildInputs = [
+    scfbuild
+  ];
+
+  buildPhase = ''
+    # Bash reimplementation of helpers/export-svg-font.js
```

Comment:
Upstream no longer has helpers/export-svg-font.js <https://github.com/hfg-gmuend/openmoji/tree/12.4.0/helpers>. Also, if possible, I recommend patching source instead of copying-pasting script contents. That is less brittle.

It looks like we are now supposed to run `npm run generate-font`: <https://github.com/hfg-gmuend/openmoji/tree/12.4.0/helpers#big-helpers>.

Reply:
I didn't copypaste, I reimplemented it entirely. The reason why I did that instead of using their scripts is that then we'd have to first package the node stuff too (and I see there's some recursive node-bash-_docker_ calls :-/ ), which is a pain.

Reply:
> Upstream no longer has helpers/export-svg-font.js

It was [moved](https://github.com/hfg-gmuend/openmoji/commit/3811fc0a9518db8f53453a8e7f341b7bd29dd152) to [`helpers/generate-font-glyphs.js`](https://github.com/hfg-gmuend/openmoji/blob/3811fc0a9518db8f53453a8e7f341b7bd29dd152/helpers/generate-font-glyphs.js).

Reply:
@fgaz I believe we can install node dependencies with `node2nix` and patch out the `docker` calls, either with `sed` or `patches`.